controllers/node.go

```diff
@@ -753,19 +749,16 @@ func deleteNode(w http.ResponseWriter, r *http.Request) {
 	}
 	go func(deletedNode *models.Node, fromNode bool) { // notify of peer change
 		var err error
-		if fromNode {
-			err = mq.PublishDeletedNodePeerUpdate(deletedNode)
-		} else {
-			err = mq.PublishPeerUpdate()
-		}
+		host, err := logic.GetHost(node.HostID.String())
 		if err != nil {
-			logger.Log(1, "error publishing peer update ", err.Error())
+			logger.Log(1, "failed to retrieve host for node", node.ID.String(), err.Error())
```

Comment:
The flow should probably stop in case of any err?

Reply:
it shouldn't stop from publishing other msgs

Reply:
If `GetHost` returns `err`, Id assume `host` is wrong, yet it still gets passed to `BroadCastDelPeer(host`.

Is that also correct?